core/gesture.ts

```diff
@@ -964,13 +856,7 @@ export class Gesture {
    * modify only this code. */
 
   /** Execute a bubble click. */
-  private doBubbleClick() {
-    // TODO (#1673): Consistent handling of single clicks.
-    if (this.startBubble instanceof WorkspaceCommentSvg) {
-      this.startBubble.setFocus();
-      this.startBubble.select();
-    }
-  }
+  private doBubbleClick() {}
```

Comment:
Delete? Or are you going to reimplement this in another PR?

Reply:
Deleted